scripts/comms_ship.lua

```diff
@@ -182,7 +182,7 @@ end
 -- @tparam ShipTemplateBasedObject ship the ship
 -- @treturn string the report
 function getStatusReport(ship)
-    local msg = "Hull: " .. math.floor(ship:getHull() / ship:getHullMax() * 100) .. "%\n"
+    local msg = string.format(_("Hull: %d%\n"), math.floor(ship:getHull() / ship:getHullMax() * 100))
```

Comment:
Is this right? There were already string.format strings in this file, but some were not.

Reply:
And what is the right way to add several variables? Like line201 :
msg = msg .. missile_type .. " Missiles: " .. math.floor(ship:getWeaponStorage(missile_type)) .. "/" .. math.floor(ship:getWeaponStorageMax(missile_type)) .. "\n"